src/wujihandpy/__init__.py

```
@@ -1,6 +1,23 @@
 from __future__ import annotations
 
-from ._core import *
+from ._core import Finger, Hand, IController, Joint, filter, logging
```

Ruff (0.15.12)

[error] Import `filter` is shadowing a Python builtin

(A004)